omniscidb/QueryEngine/CostModel/Dispatchers/ExecutionPolicy.h

```diff
@@ -27,20 +27,52 @@ struct SchedulingAssignment {
 };
 
 class ExecutionPolicy {
+  // Probe/modify modes during kernel building (do not iterate). These are the default
+  // modes.
+  const std::map<ExecutorDeviceType, ExecutorDispatchMode> devices_dispatch_modes_{
+      {ExecutorDeviceType::CPU, ExecutorDispatchMode::KernelPerFragment},
+      {ExecutorDeviceType::GPU, ExecutorDispatchMode::KernelPerFragment}};
```

Comment:
Since these are set in constructors it seems there is no reason to have the default values here (unless we use default constructors somewhere which is likely incorrect). The comment together with `const` is confusing.